test/integration/features/basic.feature

```diff
@@ -32,12 +32,12 @@ Checks whether CRC top-level commands behave correctly.
   Scenario: CRC start
     When executing "crc start -b ~/Downloads/crc_libvirt_4.1.0.tar.xz" succeeds
     Then stdout should contain "Creating VM"
-    And stdout should contain "Running"
+    And stdout should contain "CodeReady Containers instance is running"
     
   Scenario: CRC stop
     When executing "crc stop -f" succeeds
-    Then stdout should contain "true"
+    Then stdout should contain "CodeReady Containers instance stopped"
```

Comment:
Since you updated the `stop -f` message it is now ` CodeReady Containers instance forcibly stopped` which need to be updated here also. 

Another thing is this message kind of bug me since if stop is successful then you will not need any force action and the message will be `CodeReady Containers instance stopped` so how about just putting `stopped` instead of whole sentence?

Reply:
It's `CodeReady Containers instance killed` which was changed to `... forcibly stopped`. When the `stop`command is successful without user intervention, it will print this message (`... stopped`), which hasn't been changed in the latest revision.
I did not get why you think a full message is not good when `stop` is successful ?

Reply:
@cfergeau we are using  `crc stop -f` and end message for this command can be `CodeReady Containers instance forcibly stopped` or `CodeReady Containers instance stopped` which will make this test always flaky if we compare with only one message.

Reply:
@cfergeau created a followup issue https://github.com/code-ready/crc/issues/259 here.